dropwizard-auth/src/main/java/io/dropwizard/auth/Demo.java

```diff
@@ -0,0 +1,32 @@
+public class Demo {
+   public static void main(String[] args) {
+      String rpository = "test";	
+      int week = 4;
+      String day;
+	        
+      switch (week) {
```

Comment:
RSPEC-131: Switch without default case
<!-- lookout footnote separator -->
_Comment made by the analyzer sonar-check._